onnxruntime/test/python/quantization/test_op_gemm.py

```diff
@@ -318,7 +318,7 @@ def test_quantize_gemm(self):
             weight_type=QuantType.QUInt8,
         )
 
-    @unittest.skip(reason="Shape inference bug, see onnx PR #6049")
```

Comment:
Do these test-cases still fail? Is there some other bug/issue to be fixed?

Reply:
They don't fail locally if you use an onnx version with the patch. However, we still can't enable the tests because some of our CIs pip install onnx-1.16.0 directly, which doesn't include the fix.

Reply:
Some of our CIs pip install a patched onnx (Windows CIs), but others (Linux CIs) still install onnx 1.16.0 directly. We would need to think about consolidating this in the future.